.gitignore

```diff
@@ -6,3 +6,6 @@ build*
 *.bak
 src/mlpack/core/util/gitversion.hpp
 src/mlpack/core/util/arma_config.hpp
+
+.idea
```

Comment:
Ok, I'll remove both .idea and other editor-specific swap files from .gitignore then.